graylog2-rest-client/src/main/java/org/graylog2/restclient/models/api/requests/dashboards/WidgetPositionRequest.java

```diff
@@ -16,13 +16,12 @@
  */
 package org.graylog2.restclient.models.api.requests.dashboards;
 
-/**
- * @author Lennart Koopmann <lennart@torch.sh>
- */
 public class WidgetPositionRequest {
```

Comment:
Would it make sense to consolidate this class with `org.graylog2.rest.resources.dashboards.requests.org.graylog2.rest.resources.dashboards.requests` and move it into graylog2-rest-models?